WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/DashboardGoogleDomainsCardCell.swift

```diff
@@ -0,0 +1,48 @@
+import UIKit
+import SwiftUI
+
+final class DashboardGoogleDomainsCardCell: DashboardCollectionViewCell {
+    private let frameView = BlogDashboardCardFrameView()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupView()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func configure(blog: Blog, viewController: BlogDashboardViewController?, apiResponse: BlogDashboardRemoteEntity?) {
+        // TODO: Implement update configuration
+    }
+
+    private func setupView() {
+        let hostingController = UIHostingController(rootView: DashboardGoogleDomainsCardView())
+        guard let cardView = hostingController.view else {
+            return
+        }
+
+        frameView.setTitle(Strings.cardTitle)
+        frameView.add(subview: cardView)
+        frameView.onEllipsisButtonTap = { }
+        frameView.ellipsisButton.showsMenuAsPrimaryAction = true
+        // TODO: Assign menu
+        // frameView.ellipsisButton.menu = contextMenu
+
+        cardView.backgroundColor = .clear
+        frameView.translatesAutoresizingMaskIntoConstraints = false
+        contentView.addSubview(frameView)
+        contentView.pinSubviewToAllEdges(frameView, priority: .defaultHigh)
```

Comment:
❓  I read in multiple blog posts ( this for [example](https://sarunw.com/posts/swiftui-view-as-uiview/) ) that the proper way to use SwiftUI view in UIKit is by adding the `hostingController` as the parent view controller's child view controller.

I couldn't find a reason why it's a general practice to add `hostingController` as a child, but I asked ChatGPT and this is their reply:

<img width="846" alt="CleanShot 2023-08-04 at 00 07 38@2x" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/bf99e559-9283-4000-9cf7-7a2b6643b12c">

Reply:
Ah, I didn't know the reason for that. Thanks for sharing it. I'll update it accordingly.